egs2/jsut/asr1/local/data.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+SECONDS=0
+
+stage=-1
+stop_stage=1
+fs=48000
+
+db_root=downloads
```

Comment:
Would you change this part to refer ${JSUT} in db.s as following?

https://github.com/espnet/espnet/blob/ca32957c0d44e1a5bffbc5afec150c18c1d59595/egs2/vivos/asr1/local/data.sh#L28

We can specify where the corpus path is placed on.

( Sorry for many timies, I noticed just now)